PyInstaller/hooks/hook-_tkinter.py

```diff
@@ -180,7 +180,7 @@ def _find_tcl_tk(hook_api):
 
         # _tkinter depends on Tcl/Tk compiled as frameworks.
         path_to_tcl = bins[0][1]
-        if 'Library/Frameworks' in path_to_tcl:
+        if 'Library/Frameworks' in path_to_tcl and 'Python' not in path_to_tcl:
```

Comment:
```suggestion
        # Prefer framework if available, but do not use 
        # /Library/Frameworks/Python.framework/
        if ('/Library/Frameworks' in path_to_tcl and
            '/Library/Frameworks/Python.framework/' not in path_to_tcl):
```

on the other hand: Why not simply use
```suggestion
        if '/Library/Frameworks/Tcl.framework/' in path_to_tcl:
```

And related: Using Python 3.7, does your code pick up *any* framework here?

Reply:
I wanted to emphasize the fact that this was a Python lib, but you are right, your second version directly trying the path against `/Library/Frameworks/Tcl.framework/` is better.

What do you mean by _pick up any framework here_? With the current change, it does not use the Tcl framework of the system, but the tcl libraries included inside the Python framework if that is what you refer to.

Reply:
@AmbroiseGrau Sorry for not answering earlier. My last question (about  "any framework") was misleading. Please ignore.

Okay to commit my second change? Then please go ahead and we -at last- can finish this one.